public/css/mixin/progress-bar.less

```
@@ -71,8 +66,9 @@
       }
 
       &.right {
-        margin-left: -1.25em;
+        margin-left: 2.75em;
         margin-right: 0;
+        width: 8em;
```

We need neither of those. Apply this instead:

```suggestion
        // This is (.675em (:before placement) + .5em (half :before width)) + 1px (:before border)
        transform: translate(calc(-1.175em - 1px), 0);
```